requirements.txt

```diff
@@ -1 +1 @@
-numpy>=1.5
```

Comment:
This was actually here from literally the first commit (https://github.com/nengo/nengo/commit/a21c503940d4331f1876e2295554cea06816d89c#diff-b4ef698db8ca845e5845c4618278f29a) so I think it was a pretty arbitrary.